contracts/Core/BlockManager.sol

```diff
@@ -76,34 +72,19 @@ contract BlockManager is Initializable, ACL, BlockStorage {
         return(_blockMedians);
     }
 
-    function getLowerCutoffs(uint256 epoch) external view returns(uint256[] memory _lowerCutoffs) {
```

Comment:
Hey, We also have to remove these from IBlockManager, In PR https://github.com/razor-network/contracts/pull/206 it is removed as well, Either you remove it in this PR or me at end, we just have to make sure at last it goes correct

Reply:
@abhishekvispute you can cleanup in #206 .